flavors/README.md

```diff
@@ -35,6 +35,12 @@ We provide three flavors for different Exasol versions. The flavors only specify
   - Available languages: Java 11, Python 2.7, Python 3.6 and R 3.4
   - [Here](standard-EXASOL-6.2.0/FLAVOR_DESCRIPTION.md) you can find more details about the flavor, such as the provided packages.
 
+- python-3.6-minimal-6.2.0
```

Comment:
It gets its own section, as the r-4.0.minimal also got its own section. We have so to speak, one section per flavor family